index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html><html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+<title>Nicholas Mutale'Portfolio</title>
+<script src="https://kit.fontawesome.com/6dc26eb6e2.js" crossorigin="anonymous"></script>    
+</head>
+   <body>
+       <header class="nav">
+           <a class="logo" href="#">Nicholas</a>
+           <nav>
+            <i class="fas fa-bars bars"></i>
+        </nav>
+       </header>
+    <div class="header">
+            <h1>Nicholas Mutale</h1> 
+        <h2>Software Developer</h2>
+        <p>I can help you build a product , feature or 
+            website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+            <hr class="line">
+            <div class="software">
+            <ul>
+                <li class="blue1">Language</li>
+                <li class="common-li">Javascript</li>
+                <li class="common-li">Ruby</li>
+                <li class="common-li">Html</li>
+                <li class="common-li">Css</li>
+            </ul>
+            <ul>
+                <li class="blue1">Frameworks</li>
+                <li class="common-li">Bootstrap</li>
+                <li class="common-li">Ruby on Rails</li>
+                <li class="common-li">RSPec</li>
+                <li class="common-li">CapyBara</li>
+                <li class="common-li">Selenium</li>
+            </ul>
+            <ul>
+                <li class="blue1">Skills</li>
+                <li class="common-li">Codekit</li>
+                <li class="common-li">Github</li>
+                <li class="common-li">Codepen</li>
+                <li class="common-li">Gitlab</li>
+                <li class="common-li">Terminal</li>
+            </ul>
+            </div>
+            <hr class="line">
+            <ul class="icons">
+                <li><a href="#"><i class="fa fa-github bars"></i></a></li>
+                <li><a href="#"><i class="fa fa-twitter bars"></i></a></li>
+                <li><a href="#"><i class="fa fa-linkedin bars"></i></a></li>
+                <li><a href="#"><i class="fa fa-angellist bars"></i></a></li>
+                <li><a href="#"><i class="fa fa-medium bars"></i></a></li>
+            </ul>
+        </div>      
```

Comment:
- [ ] [OPTIONAL] These entire codes should be wrapped inside a `section` tag. The reason I because it gives a clear understanding to other readers and is a correct semantic tag to use.  I suggest you replace the `div` with a `section` tag.

- [ ] [OPTIONAL] Your design alignment is a bit off from the actual chosen design. The main title and subtitle need to be at the centre of the screen. And your left and right margins are too much, which is why your `p` text is a bit squeezed up. Please try to fix these issues.

Your Design | Figma Design |
:---------------:|:-------------------:|
![f1](https://user-images.githubusercontent.com/39523312/148522013-9a2e22a1-2954-4aa1-a679-80bd541fe36c.png) |![f2](https://user-images.githubusercontent.com/39523312/148522017-1b6fe68a-8634-4e6e-ba7a-98721e5a8e1e.png)